common/persistence/dataInterfaces.go

```diff
@@ -249,6 +251,18 @@ type (
 		ExpirationTime     time.Time
 		MaximumAttempts    int32
 		NonRetriableErrors []string
+		// events V2 related
+		EventStoreVersion   int32
+		CurrentResetVersion int32
```

Comment:
Do we need to include this reset version on all transfer/timer tasks? Also I think we this on history events too for cross dc replication.

Reply:
Probably yes. Thanks for reminding. I will think about it in next PR for reset